CHANGELOG.md

```diff
@@ -6,26 +6,39 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.10.0 (2023-02-16)
+
+This release contains new features and bug fixes since the 2.9.3 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+* Joins in continuous aggregates
+
```

Comment:
```suggestion
* Re-architecture of how compression works: ~2x improvement on INSERT rate into compressed chunks. 
* Full PostgreSQL 15 support for all existing features. Support for the newly introduced MERGE command on hypertables will be added on the next release. 

**PostgreSQL 12 deprecation announcement**
We will continue supporting PostgreSQL 12 until July 2023. Sooner to that time, we will announce the specific version of TimescaleDB in which PostgreSQL 12 support will not be included going forward.

**Old format of Continuous Aggregates deprecation announcement**
TimescaleDB 2.7 introduced a new format for continuous aggregates that improves performance.
All instances with Continuous Aggregates using the old format should [migrate to the new format](https://docs.timescale.com/api/latest/continuous-aggregates/cagg_migrate/) by July 2023, 
when support for the old format will be removed. 
Sooner to that time, we will announce the specific version of TimescaleDB in which support for this feature will not be included going forward.

```

Reply:
@jfjoly @fabriziomello @horzsolt kind request for a quick check on the above and a thumbs if you agree so that we can continue with merging this PR

Reply:
`Support for the newly introduced MERGE command on hypertables will be added on the next release.`

Do you think we should mention this? I know we are nearly there to finalize this feature, but it is still in review and receiving comments. Also, saying the `next release` might be misleading, I'm afraid reading this there will be customer who will expect this feature in 2.10.1.

Reply:
@horzsolt Yeah I will update to `the next minor release`

I wanted this there otherwise `Full PostgreSQL 15 support` would be misleading. What do you think?

Reply:
Ahh, I see. Maybe we can say `PostgreSQL 15 is backward compatible with the existing TimescaleDB features`, or something similar?

Reply:
I agree with everything discussed here!

Reply:
The current version looks great, thanks for updating the changelog 🚀